src/lib.rs

```diff
@@ -86,59 +86,43 @@ pub fn parse_recv_msg(buf: &[u8]) -> Sk8brdMsg {
     msg
 }
 
-pub fn console_print(buf: &[u8], len: u16) {
-    print!("{}", String::from_utf8_lossy(&buf[..len as usize]));
+pub fn console_print(buf: &[u8]) {
+    print!("{}", String::from_utf8_lossy(buf));
```

Comment:
Probably don't need an allocation here if you're willing to return on invalid UTF-8?

Reply:
Well, it's console. So it _can_ output invalid UTF-8.

Reply:
You're loosing that though, as `from_utf8_lossy()` replaces invalid UTF-8.